### DISCUSSION TITLE
Once installed, activate. A new section in the network admin area will appear called “Disciple Tools”.

### DISCUSSION THREAD

**Discussion by ianebenson** [DiscipleTools/disciple-tools-multisite#6] [Q&A]
I am sorry that I do not understand what the " the network admin area" is, so cannot see the "new section" called "Disciple Tools".

**Comment by corsacca** [comment]
Hey @ianebenson,  the Network Admin area is a wordpress feature.
If you are on a multisite and you are a super admin you can access it at the top right menu of your site.

For more info: https://wordpress.org/support/article/network-admin/

![image](https://user-images.githubusercontent.com/24901539/112628418-2b8bf180-8e33-11eb-825c-21af9238b77d.png)
